magefile.go

```diff
@@ -85,7 +85,7 @@ func Verify() error {
 	}
 
 	fmt.Println("Running external dependency checks...")
-	if err := mage.VerifyDeps("v0.3.0", "", "", true); err != nil {
+	if err := mage.VerifyDeps("v0.4.3", "", "", true); err != nil {
```

Comment:
we can drop this, and use what is defined in the release-utils

then we will need to bump the dependency as well

Reply:
@cpanato updated module and magefile, please take another look